pkg/autodiscover/autodiscover.go

```diff
@@ -104,6 +115,12 @@ func DoAutoDiscover() DiscoveredTestData {
 	data.DebugPods = findPodsByLabel(oc.Coreclient, debugLabels, debugNS)
 	data.Crds = FindTestCrdNames(data.TestData.CrdFilters)
 	data.Csvs = findOperatorsByLabel(oc.OlmClient, []configuration.Label{{Name: tnfCsvTargetLabelName, Prefix: tnfLabelPrefix, Value: tnfCsvTargetLabelValue}}, data.TestData.TargetNameSpaces)
+	data.Subscriptions = findSubscriptions(oc.OlmClient, []configuration.Label{{Name: tnfCsvTargetLabelName, Prefix: tnfLabelPrefix, Value: tnfCsvTargetLabelValue}}, data.TestData.TargetNameSpaces)
+	data.HelmList = getHelmList(oc.RestConfig, data.TestData.TargetNameSpaces)
+	openshiftVersion, _ := getOpenshiftVersion(oc.OClient)
```

Comment:
check the returned error.

Reply:
Should this be fatal? I would just print an error an put an empty string